spec/helpers/application_helper_spec.rb

```diff
@@ -162,7 +162,7 @@
 
       it "adds a list item" do
         allow(category).to receive(:items).and_return([items])
-        expect(helper.get_items(category)).to include("<li class=\"mt-0\">")
+        expect(helper.get_items(category)).to include("Charles Samuel Addams Library")
```

Comment:
It feels like this should be building.label rather than the actual value. If the factory changes, then the value will change and the test will fail. That would keep changes in one place only without consequence.

Reply:
I updated this one.